gulpfile.js

```diff
@@ -1,27 +1,34 @@
-/******************************************************
- * Building the sass
-******************************************************/
-
 var gulp = require('gulp'),
-  path = require('path'),
-  sass = require('gulp-sass'),
-  config = require('./config.json');
+path = require('path'),
+sass = require('gulp-sass'),
+config = require('./config.json');
 
 function resolvePath(pathInput) {
-  return path.resolve(pathInput).replace(/\\/g, '/');
+return path.resolve(pathInput).replace(/\\/g, '/');
 }
 
 gulp.task('build-sass', function () {
-  return (gulp
-    .src('**/**.scss', { cwd: resolvePath(paths().source.scss) })
-    .pipe(
-      sass({
-        includePaths: ['node_modules/susy/sass']
-      }))
-    .pipe(gulp.dest(paths().dest.css))
-  );
+return (gulp
+  .src('**/*.scss', { cwd: resolvePath(paths().source.scss) })
+  .pipe(
+    sass({
+      includePaths: ['node_modules/susy/sass']
+    }))
+  .pipe(gulp.dest(paths().dest.css))
+);
 });
 
 function paths() {
-  return config.paths;
+return config.paths;
 }
+
+var pug = require('gulp-pug');
+
+gulp.task('views', function buildHTML() {
+return gulp.src('./views/*.pug')
+.pipe(pug({
+pretty: true
+}))
+.pipe(gulp.dest('./html'));
+});
+
```

Comment:
This file needs some love: I see incorrect indention, unnecessary characters and missing characters